autotest/ogr/ogr_dwg.py

```diff
@@ -145,3 +145,27 @@ def test_ogr_dwg_3():
 
         ds = None
 
+
+def test_ogr_dwg_4():
+
+    with gdaltest.config_options({'DWG_INLINE_BLOCKS':'FALSE','DWG_ATTRIBUTES':'TRUE'}):
+
+        ds = gdal.OpenEx('data/cad/Building_A_Floor_0_Mapwize.dwg', allowed_drivers=['DWG'])
+
+        assert ds is not None
+
+        layer = ds.GetLayer( 'entities' )
+        defn = layer.GetLayerDefn()
+        
+        assert defn.GetFieldCount() == 28, \
+            ('did not get expected number of fields in defn. got %d'
+                                % defn.GetFieldsCount())
+                                
+        assert defn.GetFieldIndex('AVAILABILITY') >-1, \
+            ('did not get the expected field in defn. AVAILABILITY')
+        assert defn.GetFieldIndex('ROOM') >-1, \
+            ('did not get the expected field in defn. ROOM')
```

Comment:
perhaps also test the value of those attributes on a feature ?

feat = lyr.GetNextFeature()
assert feat['AVAILABILITY'] == something

Reply:
your're right, i'll make a new test

Reply:
done on 8cf4b87f21e0e74fe27cb6b62a669ca73da91c94